MultiTarget/PackageReferences/WinAppSdk.props

```diff
@@ -1,10 +1,10 @@
 <Project>
   <ItemGroup>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.240829007" />
-    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
-    <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2792.45" PrivateAssets="all" />
+    <PackageReference Condition="'$(HasWinUI)' == 'true'" Include="Microsoft.WindowsAppSDK" Version="1.6.240829007" />
+    <PackageReference Condition="'$(HasWinUI)' == 'true'" Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
```

Comment:
This is not needed, it's already a transitive NuGet dependency from WinAppSDK.

Reply:
The `PrivateAssets="All"` here isn't the default for the transitive reference, and is the fix for https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/216

Reply:
I guess we can fix that in a separate PR, but it's wrong. We should not leak that explicit package dependency.